### DISCUSSION TITLE
Building locations in lon lat

### DISCUSSION THREAD

**Discussion by seechuan** [orbisgis/geoclimate#844] [General]
While running the example in GeoClimate 0.0.1 documentation, may I know which of the generated geojson file contains the estimated building_height information? Do I derive it from the volume/area property located in the block_indicators.geojson file? How do I obtain the lon lat of the building boundaries? It seems they have been changed to some unknown x y coordinates.

**Comment by j3r3m1** [comment]
You should run GeoClimate 0.0.2 snapshot version: https://nightly.link/orbisgis/geoclimate/workflows/CI_snapshot/master/geoclimate-0.0.2-SNAPSHOT.zip

**Reply by seechuan** [reply]
Is there a sample OSM workflow file? I used the 0.01 workflow and it says "Cannot execute the OSM workflow".

**Reply by j3r3m1** [reply]
The wiki is your friend ! =)
Here is the Windows link, you should be able to find the linux one if needed or change it to you os path file:
https://github.com/orbisgis/geoclimate/wiki/Default-case-with-OSM-for-Windows#create-and-understand-the-configuration-file
